### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by Robert-Steiner** [nbigaouette/onnxruntime-rs#111]
Hi there,

I would be interested to know what the status of this project is. Are there any plans to further develop this library? For example by reviewing/accepting PRs?

**Comment by ia0** [comment]
By the way, for those interested in an alternative, there is https://crates.io/crates/ort.

**Comment by hmeine** [comment]
~Unfortunately, ort is not an alternative, but a higher-level API that depends on this very crate.~

EDIT: Disregard this. I was following outdated or wrong information from some older forum posts.

**Comment by ia0** [comment]
> Unfortunately, ort is not an alternative, but a higher-level API that depends on this very crate.

Where do you see that `ort` depends on this crate?

Regarding the high-level aspect, I'm also not following. Both projects have a high- and low-level crate.

**Comment by hmeine** [comment]
Sorry, I was trying to get an overview myself and gathered information from Reddit and other sources, maybe also some LLM. There were so many crates being mentioned (`onnxruntime-sys`, `onnxruntime-rs`, `onnxruntime`, `ort` at the very least) that I became confused and I wanted to help others when I *thought* I understood some relation. But apparently I was wrong.

In the end, I found that the official onnxruntime repository by Microsoft now contains Rust bindings, which is what one gets from `cargo add onnxruntime` (https://docs.rs/onnxruntime/latest/onnxruntime/).

Now I hope that this time I leave this issue behind with more helpful and up-to-date information.

**Comment by ia0** [comment]
I agree it's confusing because someone copied this project to the Microsoft repository (with the same name). But both versions are unmaintained since 3+ years. The Microsoft copy was actually never published. And I wouldn't call it official (it wasn't written by Microsoft). One could create a PR to delete it to reduce confusion.
